### PR TITLE
Kpn logo

### PR DESCRIPTION
**Pull request by rpartapsing**

### CODE REVIEW

File: REPO [hunk 1]
```diff
@@ -0,0 +1 @@
+registry.cp.kpn-dsh.com/cp/cachet
```
Comment (svyotov) [inline REPO:1]: missing `\n`

File: VERSION [hunk 2]
```diff
@@ -0,0 +1 @@
+2.4.1
```
Comment (svyotov) [inline VERSION:1]: missing `\n`

File: REPO (hunk 1 above)
Comment (rpartapsing) [reply REPO:1]: check

File: VERSION (hunk 2 above)
Comment (rpartapsing) [reply VERSION:1]: check

**Pull request merged**